pkgs/sage-conf/.gitignore

```diff
@@ -1,4 +1,6 @@
 /_sage_conf/_conf.py
+/setup.cfg
+/pyproject.toml
 /build
```

Comment:
`setup.cfg` is deleted. Then why here?

Reply:
Likely I had a branch where this `setup.cfg` was generated from a `setup.cfg.m4`, and this reduces noise when working on a branch. As long as no git-controlled files are listed in the .gitignore, this is harmless.

Reply:
OK. No big deal.